src/main/rx/crsf.c

```diff
@@ -165,7 +175,6 @@ STATIC_UNIT_TESTED uint8_t crsfFrameStatus(void)
             if (crc != crsfFrame.frame.payload[CRSF_FRAME_RC_CHANNELS_PAYLOAD_SIZE]) {
                 return RX_FRAME_PENDING;
             }
-            crsfFrame.frame.frameLength = CRSF_FRAME_RC_CHANNELS_PAYLOAD_SIZE + CRSF_FRAME_LENGTH_TYPE_CRC;
```

Comment:
Why has this line been removed?

Reply:
I couldn't find anywhere it was being used in the code below.  I can re-add it, but it doesn't appear to have a purpose.

Reply:
`crsfFrame.frame.frameLength` is used in the `crsfFrameCRC`. So it should actually be set before calling `crsfFrameCRC`. Presumably it only works currently because it is constant and static, so as long as it is set once `crsfFrameCRC` will calculate properly.

As a general principle you shouldn't remove code to "tidy" things in a PR that introduces a new feature. Suppose that it was required, then the only way to back out the change would be to back out this entire PR and so loose the feature when fixing the bug.

By all means make changes that tidy things up, but they should be in their own PR.

Reply:
Roger that.  Would you like me to restore the previous line?

Reply:
crsfFrame.frame.frameLength is part of received message, so it should be either correct (and there is no need to set it) or it does not match type and frame should be discarded/handled according to length.

Reply:
can we handle some of this frameLength checking logic that you propose in a different PR?  I don't mind adding a change for it, it just seems a bit outside of the scope of the issue this PR resolves.

Reply:
Another thought, though -- wouldn't the CRC check be enough to validate the contents of the frame?  Is the length check even required?

Reply:
For the sake of scope, this change has been restored.  No sense in changing something that isn't part of the issue being resolved.

Reply:
Sorry, I wasn't thinking straight here. @ledvinap is correct about this. So this should indeed be removed - but this should be done in a separate PR.